src/readii/data/label.py

```
@@ -0,0 +1,308 @@
+            # Create a dictionary to map event values to numeric values
+            event_column_value_mapping = survivalStatusToNumericMapping(event_outcome_column=dataframe_with_standardized_outcome[outcome_column_label])
+
+        else:
+            # Convert all dictionary keys in provided mapping to lowercase
+            event_column_value_mapping = dict((status.lower(), value) for status, value in event_column_value_mapping.items())
+
+            # Check if user provided dictionary handles all event values in the outcome column
+            if set(existing_event_values) != set(event_column_value_mapping.keys()):
+                msg = f"Not all event values in {outcome_column_label} are handled by the provided event_column_value_mapping dictionary."
+                logger.exception(msg)
+                raise e
```

Ruff (0.8.2)

Undefined name `e`

(F821)